ui/app/AppLayouts/Wallet/panels/WalletHeader.qml

```diff
@@ -60,12 +60,12 @@ Item {
 
         StatusAddressPanel {
             objectName: "addressPanel"
-            address: currentAccount.mixedcaseAddress
-
+            address: currentAccount.ens || currentAccount.mixedcaseAddress
```

Comment:
Maybe we should change that property's name to `title` or something, since it's not always an address now.

Reply:
me neither :) but I will check